apps/publish.sh

```diff
@@ -36,7 +36,8 @@ MACHINES=${MACHINES-""}
 PLATFORMS=${MANIFEST_PLATFORMS_DEFAULT-""}
 
 APP_IMAGES_ROOT_DIR="${APP_IMAGES_ROOT_DIR-/var/cache/bitbake/app-images}"
-OSTREE_REPO_DIR="${APP_IMAGES_ROOT_DIR}/ostree-repo"
+#OSTREE_REPO_DIR="${APP_IMAGES_ROOT_DIR}/ostree-repo"
+OSTREE_REPO_DIR="${OSTREE_REPO_DIR-$(mktemp -d -p ${HOME})}"
```

Comment:
can just delete this commented out line. I assume using a local volume speeds things up because it skips overlayfs or something?

Reply:
${APP_IMAGES_ROOT_DIR}/ostree-repo refers to NFS and committing to an ostree repo located on NFS is veryyy slow.
Sure, I will remove it.
OSTREE_REPO_DIR="${OSTREE_REPO_DIR-$(mktemp -d -p ${HOME})}" - this actually points to container write layer managed by overlayfs, it doesn't cause much trouble, but ideally, it would be great if jobserve worker adds `--tmpfs` so the code running in a container could store something directly on a host filesystem. In this case, I could do "checkout with hardlinks", fetch and then commit back with `--link-checkout-speedup` which helps to speed up the whole process.